spec/jobs/signal_adapter/receive_polling_job_spec.rb

```diff
@@ -187,10 +182,8 @@
     describe 'given a contributor who has unsubscribed and requests to resubscribe',
              vcr: { cassette_name: :receive_signal_message_to_resubscribe } do
       before do
-        unless Setting.signal_server_phone_number
```

Comment:
This looks intentional, why was this put in place? Did I miss sth. @mattwr18 ?

Reply:
@roschaefer we already had a conversation about this code in another PR; unfortunately, I can't find it. It was actually you that added this code in https://github.com/tactilenews/100eyes/commit/5845dc4ecefd1bc0ea6538733aa7ddee9a52b4d5... You did not go into great detail why you added it, but the commit message states that you were fixing ci. It then has been refactored and copy/pasted throughout the spec file.

I can say for my part that I find the code useful the first time I record the cassette so that I can get some actual data back when I run the receive messages rake task. After the cassette is recorded, I think we can and probably should delete the code.

I think just having the line:

```
allow(Setting).to receive(:signal_server_phone_number).and_return('SIGNAL_SERVER_PHONE_NUMBER')
```

was enough to get the ci to pass 2 years ago, but I think you maybe wanted to continue using what you had configured in `.env` :shrug:

Reply:
As discussed in our pairing, it's too confusing, so let's just remove it. Maybe we can check if we're in `record` mode instead of checking the configuration.